Add an option to sort year dropdown ascending or descending

cflinchbaugh opened the pull request:
Added support for optionally sorting the rendered title's Years from max to min via the new yearOrderMinMax boolean value (defaulted to **true** to maintain original behavior), updated the README.md to include option definition

cflinchbaugh commented:
Linking for convenience:
#773

wojciechczerniak commented on README.md:
```diff
@@ -135,6 +135,7 @@ Pikaday has many useful options:
 * `firstDay` first day of the week (0: Sunday, 1: Monday, etc)
 * `minDate` the minimum/earliest date that can be selected (this should be a native Date object - e.g. `new Date()` or `moment().toDate()`)
 * `maxDate` the maximum/latest date that can be selected (this should be a native Date object - e.g. `new Date()` or `moment().toDate()`)
+* `yearOrderMinMax` the sorted order of the years displayed in the dropdown, defaults to true
```
I would recommend option name `yearOrder` nad value `ascending` / `descending`.

cflinchbaugh replied:
Please refer to [pull request 833](https://github.com/Pikaday/Pikaday/pull/833).

wojciechczerniak commented on pikaday.js:
```diff
@@ -407,11 +409,20 @@
+        } else {
+            for (arr = []; j > i && j >= opts.minYear; j--) {
+                if (j <= opts.maxYear) {
+                    arr.push('<option value="' + j + '"' + (j === year ? ' selected="selected"': '') + '>' + (j) + '</option>');
```
Can you find solution that don't repeat the `<option>` code? Now we would have to change it twice if we want to update the line.

cflinchbaugh replied:
Please refer to [Pull Request 833](https://github.com/Pikaday/Pikaday/pull/833)

cflinchbaugh commented:
I've created a new [pull request](https://github.com/Pikaday/Pikaday/pull/833) which addresses the feedback provided above.  Thanks to [wojciechczerniak](https://github.com/wojciechczerniak)!
